python/pyiceberg/cli/output.py

```diff
@@ -97,13 +97,13 @@ def describe_table(self, table: Table):
         for key, value in metadata.properties.items():
             table_properties.add_row(key, value)
 
-        schema_tree = Tree("Schema")
+        schema_tree = Tree(f"Schema, id={table.metadata.current_schema_id}")
         for field in table.schema().fields:
             schema_tree.add(str(field))
 
         snapshot_tree = Tree("Snapshots")
         for snapshot in metadata.snapshots:
-            snapshot_tree.add(f"Snapshot {snapshot.schema_id}: {snapshot.manifest_list}")
+            snapshot_tree.add(f"Snapshot {snapshot.snapshot_id}, schema {snapshot.schema_id}:  {snapshot.manifest_list}")
```

Comment:
Is the extra space needed?

Reply:
Definitely not, slipped in there. Thanks for noticing.